source/_docs/upstream-updates.md

```diff
@@ -160,6 +160,14 @@ It's important to relay the need for updating core to maintainers, even if you p
 
 Once you have communicated the issue, you can [manually apply updates from the command line](#apply-upstream-updates-manually-from-the-command-line-and-resolve-merge-conflicts).
 
+### Custom Upstream Updates Not Available

```

Comment:
I like the subheader here, but it conflicts with  [line 156](https://github.com/pantheon-systems/documentation/pull/3453/files#diff-2d8862e1a7f9fb609a84a234b6181faeR156)

That line was a messy change leftover from when I scrubbed "public distributions" from the docs. Can you take a swing at renaming [line 156](https://github.com/pantheon-systems/documentation/pull/3453/files#diff-2d8862e1a7f9fb609a84a234b6181faeR156) @alexfornuto? It's not clear that these scenarios refer to two completely different products